copier/cli.py

```diff
@@ -42,21 +66,21 @@ class CopierApp(cli.Application):
     CALL_MAIN_IF_NESTED_COMMAND = False
     data: AnyByStrDict = {}
 
-    answers_file = cli.SwitchAttr(
+    answers_file: cli.SwitchAttr = cli.SwitchAttr(
```

Comment:
I'm surprised about these changes. I'd say this is a bug on mkdocs because mypy doesn't yell about this. `cli.SwitchAttr` is a class, so it's pretty obvious that this attribute will have that class as a type.

Any ideas on this subject?

Reply:
Well, `mkdocstrings` is not as powerful as `mypy`, and is only using actual type annotations, not deducing them from the value of the attribute. So if you want the type to appear in the docs, you *have* to annotate it.

Reply:
Actually `pytkdocs` could be improved to use the attribute's type if no annotation is found. I'll open the related feature request 🙂